src/constant/index.ts

```diff
@@ -315,6 +316,7 @@ export enum WALLET_BRAND_TYPES {
   KEYSTONE = 'Keystone',
   COOLWALLET = 'CoolWallet',
   DEFIANT = 'Defiant',
+  AIRGAP = 'AIRGAP',
```

Comment:
```suggestion
  AIRGAP = 'AirGap',
```
since this will use as prefix of alias name by default